external-resolver/src/main/scala/org/genivi/sota/resolver/db/DeviceRepository.scala

```diff
@@ -90,82 +92,75 @@ object DeviceRepository {
    */
 
   // scalastyle:off
-  class InstalledPackageTable(tag: Tag) extends Table[(Namespace, Uuid, PackageId)](tag, "InstalledPackage") {
-    def device         = column[Uuid]("device_uuid")
-    def namespace      = column[Namespace]("namespace")
-    def packageName    = column[PackageId.Name]("packageName")
-    def packageVersion = column[PackageId.Version]("packageVersion")
+  class InstalledPackageTable(tag: Tag) extends Table[(Uuid, UUID)](tag, "InstalledPackage") {
+    def device = column[Uuid]("device_uuid")
+    def packageUuid = column[UUID]("package_uuid")
```

Comment:
Now we're using java.util.UUID and an AnyVal wrapping a Refined[String,Uuid].
what do you think, should we unify this in the future? I had a discussion with @stevana  about datatype specific UUID's in order to not being able to mix e.g. device UUIDs with package UUIDs.
What would you say, overkill?